docs/how-tos/howto-set-ibft.md

```diff
@@ -39,10 +39,27 @@ go run main.go ibft init test-chain-3
 go run main.go ibft init test-chain-4
 ````
 
-## Step 2: Generate an IBFT genesis file with the previous accounts as validators
+Each of these commands will print [node ID](https://docs.libp2p.io/concepts/peer-id/). You will need that information for the next step.
+
+## Step 2: Using the clients as bootstrap nodes
+
+To start the clients as bootstrap nodes you will need to specify their address schemes by encoding them
+into a [multiaddr format](https://docs.libp2p.io/concepts/addressing/):
+```
+/ipv4/<ip_address>/tcp/<port>/p2p/<node_id>
+```
+
+To do so, you should specify the IP address and the port for each of the clients,
+and append the node ID (from the previous step) to it. For example:
+
+```
+/ipv4/127.0.0.1/tcp/10001/p2p/16Uiu2HAmJxxH1tScDX2rLGSU9exnuvZKNM9SoK3v315azp68DLPW
+```
+
+## Step 3: Generate an IBFT genesis file with the previous accounts as validators
 
 ````bash
-go run main.go genesis --consensus ibft --ibft-validators-prefix-path test-chain-
+go run main.go genesis --consensus ibft --ibft-validators-prefix-path test-chain- --bootnode /ipv4/127.0.0.1/tcp/10001/p2p/<node_id_1> --bootnode /ipv4/127.0.0.1/tcp/20001/p2p/<node_id_2> --bootnode /ipv4/127.0.0.1/tcp/30001/p2p/<node_id_3> --bootnode /ipv4/127.0.0.1/tcp/40001/p2p/<node_id_4>
```

Comment:
`ip4` is correct protocol name in libp2p

```suggestion
go run main.go genesis --consensus ibft --ibft-validators-prefix-path test-chain- --bootnode /ip4/127.0.0.1/tcp/10001/p2p/<node_id_1> --bootnode /ip4/127.0.0.1/tcp/20001/p2p/<node_id_2> --bootnode /ip4/127.0.0.1/tcp/30001/p2p/<node_id_3> --bootnode /ip4/127.0.0.1/tcp/40001/p2p/<node_id_4>
```

Reply:
Omg, GJ for noticing that 🍻